docs/docsite/rst/community/contributor_path.rst

```diff
@@ -0,0 +1,104 @@
+
+Specific knowledge for code developers
+----------------------------------------
+
+Code development requires the most technical knowledge. Let's sort out what an Ansible developer should learn.
+
+
+You should understand at least *basics* of the following tools:
+
+- `Python programming language <https://docs.python.org/3/tutorial/>`_
+- `Git <https://git-scm.com/docs/gittutorial>`_
+- `GitHub collaborative development model through forks and pull requests <https://docs.github.com/en/github/collaborating-with-pull-requests/getting-started/about-collaborative-development-models>`_
+
+You can learn these tools more in depth when working on your first contributions.
+
+
+Each Ansible project has its own set of contributor guidelines. Familarize yourself with these as you prepare your first contributions.
+
+* :ref:`Ansible Core development <developer_guide>`.
+* :ref:`Ansible collectoin development <developing_collections>` and the collection-level contributor guidelines in the collection repository.
+
+
+Making your first contribution
+==============================
+
+You can find some ideas how you can contribute in :ref:`how_can_i_help` and the `collection repository <https://github.com/ansible-collections/>`_  ``README`` and ``CONTRIBUTING`` files. To make your first experience as smooth as possible, read the repository documentation carefully, then ask the repository maintainers for guidance if you have any questions.
+
+You can also look for GitHub issues labeled with the ``easyfix``, ``good_first_issue``, and ``docs`` labels.  Add a comment on the GitHub issue to say you are looking at it and to ask for help if you need it.
+
+Continue to contribute
+======================
+
+We don't expect everybody to know everything. Start small, think big. When you contribute regularly, your proficiency and judgment in the related area will improve quickly and, along with this, the importance of your presence in the project.
+
+See :ref:`communication` for ways to communicate and engage with the Ansible community, including working group meetings, accessing the Bullhorn news bulletin, and upcoming contributor summits.
+
+
+Teach others
+============
+
+Share your experience with other contributors through `improving documentation <ommunity_documentation_contributions>`, answering question from other contributors and users  on `Matrix/Libera.Chat IRC <communicate>`, giving advice in issues and pull requests, and discussing the `Community Topics <https://github.com/ansible-community/community-topics/issues>`_.
+
+Become a collection maintainer
+===============================
+
+If you are a code contributor to a collection, you can get extended permissions in the repository and become a maintainer. A collection maintainer is a contributor trusted by the community who makes significant and regular contributions to the project and showed themselves as a specialist in the related area. See :ref:`maintainers` for details.
```

Comment:
```suggestion
If you are a code contributor to a collection, you can get extended permissions in the repository and become a maintainer. A collection maintainer is a contributor trusted by the community who makes significant and regular contributions to the project and serves as a specialist in the related area. See :ref:`maintainers` for details.
```

Reply:
i would keep the sentence as is.
`showed themselves` is an important thing here.
We're trying to say that "the person's judgment and activities (like great reviews, for example) show that they are a specialist in the technical area the collection servers. So we can trust their judgment and promote them to a maintainer."

Reply:
agreed.